src/main.c

```diff
@@ -87,9 +99,28 @@ notify_cb(NotifyNotification *notification, char *action, gpointer user_data) {
                                           (gchar**)gtk_theme_names->pdata,
                                           (gchar**)icon_theme_names->pdata,
                                           (gchar**)sound_theme_names->pdata,
-                                          NULL, NULL, NULL, install_themes_cb, NULL);
+                                          NULL, NULL, NULL, install_themes_cb, state);
     }
-    g_object_unref(notification);
+}
+
+static void
+show_notification (DsState *state)
```

Comment:
Perhaps call `show_install_notification` - this isn't the only notification that is shown.